src/components/media/MediaModal.jsx

```diff
@@ -18,13 +16,6 @@ const MediaModal = ({ onClose, onProceed, type, showAltTextModal = false }) => {
 
   const methods = useForm({
     mode: "onTouched",
-    resolver: yupResolver(
-      Yup.object({
-        altText: Yup.string()
-          // .required("Alt text is required")
-          .max(100, "Alt text should be less than 100 characters"),
-      })
-    ),
   })
 
```

Comment:
sanity check - we no longer require validation for alt text?

Reply:
This has been moved into the alt text modal itself! react-hook-form doesn't allow a resolver + individual field validation to exist at the same time